pkgs/by-name/hy/hyprnotify/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, pkg-config
+, alsa-lib
+}:
+
+buildGoModule rec {
+  pname = "hyprnotify";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "codelif";
+    repo = "hyprnotify";
+    rev = "v${version}";
+    hash = "sha256-dAclsaY8gjHFyWe1/df98Tz5a1tJTZz2xvx8CZv1Q6Q=";
+  };
+  nativeBuildInputs = [
+    pkg-config
+  ];
+  buildInputs = [
+    alsa-lib
+  ];
+
+  vendorHash = "sha256-AZDtaiSNq7em876Q9f+YeDxboqVwA8IE9dDM6zggFXs=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

tracking issue: #292468

---

Don't this was fixed, btw

---

I am being yelled at when I remove that.

---

You have to put `lib.` where needed as well